python/examples/core/transform.py

```diff
@@ -134,6 +134,36 @@ def __call__(self, module: Module, fun_name: str):
     return module
 
 
+class SaveIR(Transform):
```

Comment:
Note that the codegen strategy pipeline will soon be deprecated in favor of the scheduling dialect.
@ftynse may prefer you hold off on this?

Reply:
Well if it is really a problem I can try to move it in the experimental/ folder as well. Please, let me know!

Reply:
I think it is really about reducing extra work for @ftynse he should say what he prefers.